docs/ref-info/ref-tokens.md

```diff
@@ -0,0 +1,56 @@
+---
+title: UID2 Tokens and Refresh Tokens
+description: Information for publishers about UID2 tokens and refresh tokens.
+hide_table_of_contents: false
+sidebar_position: 06
+---
+
+import Link from '@docusaurus/Link';
+
+# UID2 Tokens and Refresh Tokens
+
+When a publisher sends a user's <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) to the UID2 Operator, whether via one of the UID2 SDKs or the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint, the UID2 Operator returns a <a href="glossary-uid#gl-uid2-token">UID2 token</a> with associated values. The token is an opaque alphanumeric string, and is pseudonymous. Because of the way UID2 works, different instances of activity, on browsers, CTV, and electronic devices such as phone and tablets, can be matched without compromising the privacy of the individual. The token is designed so that it cannot be reverse engineered to arrive at the original email address or phone number.
```

Comment:
@sunnywu thx much. Made the first change. Second one: yes, by associated values, which we use in several places in the doc, we mean the refresh token, timestamp etc. that come with the token.  I adjusted to specify refresh token as an example, but I think that wording is clear.
On the other points... this will go to Legal after you,. I can call out those sections but he will check those things anyway.

Reply:
Follow up on this: these sections were changed.